src/client/Client.cc

```diff
@@ -14137,9 +14137,9 @@ int Client::_rename(Inode *fromdir, const char *fromname, Inode *todir, const ch
   }
   if (cct->_conf.get_val<bool>("client_quota") && fromdir != todir) {
     Inode *fromdir_root =
-      fromdir->quota.is_enabled(QUOTA_MAX_FILES) ? fromdir : get_quota_root(fromdir, perm, QUOTA_MAX_FILES);
+      fromdir->quota.is_enabled(QUOTA_ANY) ? fromdir : get_quota_root(fromdir, perm, QUOTA_ANY);
     Inode *todir_root =
-      todir->quota.is_enabled(QUOTA_MAX_FILES) ? todir : get_quota_root(todir, perm, QUOTA_MAX_FILES);
+      todir->quota.is_enabled(QUOTA_ANY) ? todir : get_quota_root(todir, perm, QUOTA_ANY);
```

Comment:
You could just use `todir->quota.is_enabled()` instead to make it consistent with the existing code in other places.

Reply:
True. `QUOTA_ANY` is the default of the only parameter for `quota.is_enabled()`. I'll make that change.

Reply:
This change has been added.